server/controllers/finance/fiscal.js

```diff
@@ -595,3 +597,56 @@ function accountBanlanceByTypeId() {
     ORDER BY ac.number
   `;
 }
+
+// client debts by filscal year id and date range
+function receivables() {
+  return `
+    SELECT 
```

Comment:
Can you get this from the `period_total`'s table?  It would make the query faster.